keras_nlp/layers/transformer_decoder_test.py

```diff
@@ -134,7 +134,9 @@ class MyModel(keras.Model):
             def __init__(self):
                 super().__init__()
                 self._decoder = transformer_decoder.TransformerDecoder(
-                    intermediate_dim=4, num_heads=2
+                    intermediate_dim=4,
+                    num_heads=2,
+                    has_cross_attention=True,
```

Comment:
Hmm, why do we need this actually? Won't this line at the start of call `has_encoder_sequence = encoder_sequence is not None`, mean the layer will be built with cross attention as soon as the decoder is called on two inputs?

Reply:
Sorry 🤦🏼 , not needed. Changing it back